### PR TITLE
Envoy Quickstart

### PR DESCRIPTION
**Pull request by sjyothi54**
**Ticket:** https://new-relic.atlassian.net/browse/NR-231691
**Description:** Created new Envoy quickstart

## Pre merge checklist

- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/envoy/ClusterBindErrors.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 1
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 300
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+  - priority: WARNING
```
Comment (pkudikyala) [inline alert-policies/envoy/ClusterBindErrors.yml:29]: Remove the warning condition.

File: alert-policies/envoy/ClusterUpdateFailed.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 1
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 300
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+  - priority: WARNING
```
Comment (pkudikyala) [inline alert-policies/envoy/ClusterUpdateFailed.yml:29]: Remove the warning condition.

File: alert-policies/envoy/DrainingListener.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 1
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 300
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+  - priority: WARNING
```
Comment (pkudikyala) [inline alert-policies/envoy/DrainingListener.yml:29]: Remove the warning condition.

File: quickstarts/envoy/config.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    description: |
+      Enhance the performance monitoring and instrumentation of your Envoy by integrating New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/envoy-monitoring-integration/
+keywords:
+  - envoy
+  - envoy proxy
+  - edge gateway
+  - proxy
+  - NR1_addData
```
Comment (pkudikyala) [inline quickstarts/envoy/config.yml:34]: Please add the NR1_sys keyword.

**Review by pkudikyala — CHANGES_REQUESTED** [review]
Hi @sjyothi54,
Could you please make the review changes.

**Comment by sjyothi54** [comment]
> Hi @sjyothi54, Could you please make the review changes.

@pkudikyala Made the requested changes, Kindly review it

**Review by pkudikyala — DISMISSED** [review]
LGTM!

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by pkudikyala — APPROVED** [review]
LGTM!

**Pull request merged**